<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadHeadIcon</class>
 <widget class="QWidget" name="UploadHeadIcon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="uploadImage">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border:1px solid #C3C8CC;color:black; background:white;}
QPushButton:hover{border:1px solid #C3C8CC;color:black; background:#F8F8F8;}
QPushButton:pressed{border:1px solid #C3C8CC; color:black; background:#F0F0F0;}</string>
         </property>
         <property name="text">
          <string>上传本地照片</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>350</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(:/res/headicon/CircleMask.png)</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>350</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border-image:url(:/res/headicon/bighead_editor_zoom_out_normal.png)}
QPushButton:hover{border-image:url(:/res/headicon/bighead_editor_zoom_out_hover.png)}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="styleSheet">
          <string notr="true">
  QSlider::groove:horizontal {
      height: 3px; 
      background: #BBBBBB;
  }

  QSlider::handle:horizontal {
      background: #12B7F5;
      border: 0px;
      width: 10px;
      margin: -4px 0; 
      border-radius: 5px;
  }</string>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border-image:url(:/res/headicon/bighead_editor_zoom_in_normal.png)}
QPushButton:hover{border-image:url(:/res/headicon/bighead_editor_zoom_in_hover.png)}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="leftRotate">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border-image:url(:/res/headicon/bighead_editor_rotate_left_normal.png)}
QPushButton:hover{border-image:url(:/res/headicon/bighead_editor_rotate_left_hover.png)}
QPushButton:pressed{border-image:url(:/res/headicon/bighead_editor_rotate_left_down.png)}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rightRotate">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border-image:url(:/res/headicon/bighead_editor_rotate_right_normal.png)}
QPushButton:hover{border-image:url(:/res/headicon/bighead_editor_rotate_right_hover.png)}
QPushButton:pressed{border-image:url(:/res/headicon/bighead_editor_rotate_right_down.png)}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>39</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border:0px;color:white; background:#12B7F5;}QPushButton:hover{border:0px;color:white; background:#0EABE6;}
QPushButton:pressed{border:0px;  color:white; background:#0E98CB;}</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border:1px solid #C3C8CC;color:black; background:white;}
QPushButton:hover{border:1px solid #C3C8CC;color:black; background:#F8F8F8;}
QPushButton:pressed{border:1px solid #C3C8CC; margin-top:1px; color:black; background:#EAEEF5;}</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
